src/components/common/Actions.vue

```diff
@@ -31,20 +31,19 @@
         </v-btn>
       </v-fade-transition>
 
-      <v-fade-transition hide-on-leave>
-        <v-btn id="review-confirm-btn" large color="primary"
-          :to="nextRoute"
-          v-show="isShowReviewConfirmBtn"
-          :disabled="isBusySaving"
-        >
-          <span>{{ nextButtonLabel }}</span>
-          <v-icon>mdi-chevron-right</v-icon>
-        </v-btn>
-      </v-fade-transition>
+<!--      <v-fade-transition hide-on-leave>-->
+<!--        <v-btn id="review-confirm-btn" large color="primary"-->
+<!--          :to="nextRoute"-->
+<!--          v-show="isShowReviewConfirmBtn"-->
+<!--          :disabled="isBusySaving"-->
+<!--        >-->
+<!--          <span>{{ nextButtonLabel }}</span>-->
+<!--          <v-icon>mdi-chevron-right</v-icon>-->
+<!--        </v-btn>-->
+<!--      </v-fade-transition>-->
```

Comment:
Wasn't sure of the entire flow if we will need anything beyond a File And Pay. Commented out for now, can remove later.